packages/orchestration/src/utils/start-helper.js

```diff
@@ -101,6 +105,6 @@ export const provideOrchestration = (
     vowTools,
     ...remotePowers,
   });
-  return { ...facade, chainHub, zone };
+  return { ...facade, chainHub, agNamesTools, zone };
```

Comment:
Reviewers might suggest this should be a locally provided endowment, instead of through `provideOrchestration`

Reply:
Yes please.